app/org/sagebionetworks/bridge/services/ConsentService.java

```diff
@@ -226,16 +225,14 @@ public void consentToResearch(Study study, SubpopulationGuid subpopGuid, StudyPa
         if(!withdrawSignatures(account, subpopGuid, withdrewOn)) {
             throw new EntityNotFoundException(ConsentSignature.class);
         }
+        Map<SubpopulationGuid,ConsentStatus> statuses = getConsentStatuses(context, account);
+        if (!ConsentStatus.isUserConsented(statuses)) {
+            account.setSharingScope(SharingScope.NO_SHARING);
+        }
```

Comment:
I'm not seeing the test in ConsentServiceTest (either pre-existing or new).

Reply:
Ok, added 4 explicit tests of this in the mock class.